frontend/src/features/PriorNotification/components/ManualPriorNotificationForm/fields/FormikVesselSelect.tsx

```diff
@@ -80,24 +92,24 @@ export function FormikVesselSelect({ onChange, readOnly }: FormikVesselSelectPro
         return
       }
 
-      setDefaultValue(input.value)
+      setValue(input.value)
     },
 
     // Ignore `input.value` change since it should only be called on mount.
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [setDefaultValue]
+    [setValue]
   )
 
   return (
     <Field>
       <StyledVesselSearch
-        key={key}
         baseRef={newWindowContainerRef}
-        defaultValue={defaultValueRef.current}
+        cachedVesselIdentities={cachedVesselIdentities}
```

Comment:
Je propose qu'on renomme cette props, il n'y a pas de cache.
Lorsqu'on recherche des navires, on cherche :
- dans la table `last_positions`, qui est dans redux `state.vessel.vessels`
- la table vessels, par l'API

Dans le 1er cas,on cherche dans les positions VMS et dans le deuxième cas on cherche dans un référentiel navire. Le 1er cas n'est pas un cache du 2ème cas.

Reply:
Renommé to `vesselIdentitiesFromLastPositions`.